packages/components/FocusZone/src/FocusZone.ts

```diff
@@ -1,15 +1,19 @@
+/**
+ * Copyright (c) Microsoft Corporation.
+ * Licensed under the MIT License.
+ * @format
+ */
+
 import * as React from 'react';
 import { findNodeHandle } from 'react-native';
 
-import { ensureNativeComponent } from '@fluentui-react-native/component-cache';
 import { useViewCommandFocus } from '@fluentui-react-native/interactive-hooks';
 import type { IUseStyling } from '@uifabricshared/foundation-composable';
 import { composable } from '@uifabricshared/foundation-composable';
 import { mergeSettings } from '@uifabricshared/foundation-settings';
 
 import type { FocusZoneProps, FocusZoneSlotProps, FocusZoneType } from './FocusZone.types';
-
-const RCTFocusZone = ensureNativeComponent('RCTFocusZone');
+import RCTFocusZone from './FocusZoneNativeComponent';
```

Comment:
Probably out of scope, but could we also not leak the apple 3 letter prefix convention to JS code here? 😅 I know this is long existing code.